README.md

```diff
@@ -109,7 +109,7 @@ Other dedicated linters that are built-in are:
 | [djlint][djlint]                   | `djlint`               |
 | [dotenv-linter][dotenv-linter]     | `dotenv_linter`        |
 | [editorconfig-checker][ec]         | `editorconfig-checker` |
-| [erb-lint][erb-lint]               | `erb_lint`             |
+| [erb_lint][erb_lint]               | `erb_lint`             |
```

Comment:
The link definition is still erb-lint, see https://github.com/mfussenegger/nvim-lint/blob/d44185d3e57e00c164485a454174294b36ef7067/README.md?plain=1#L449

Reply:
I guess I was confused with the formatted I use, you can ignore this one.